module/VuFind/tests/unit-tests/src/VuFindTest/Solr/WriterTest.php

```diff
@@ -45,22 +45,15 @@
  */
 class WriterTest extends \PHPUnit\Framework\TestCase
 {
-    use \VuFindTest\Feature\SearchServiceTrait;
-    use \VuFindTest\Feature\ReflectionTrait;
-
     /**
      * Test commit
      *
      * @return void
      */
     public function testCommit()
     {
-        $bm = $this->getBackendManagerWithMockSolr();
-        $connector = $bm->get('Solr')->getConnector();
-        $connector->expects($this->once())->method('write')
-            ->with($this->isInstanceOf(CommitDocument::class));
-        $writer = new Writer($this->getSearchService($bm), $this->getMockChangeTracker());
-        $writer->commit('Solr');
+        $command = new WriteDocumentCommand('Solr', new CommitDocument(), 60 * 60);
```

Comment:
Maybe it would make sense to rename all these $command variables to $expectedCommand, just to make it clearer what role they play in the test.

Reply:
Done.